pkg/cli/environment.go

```diff
@@ -108,6 +110,7 @@ func New() *EnvSettings {
 		RepositoryCache:           envOr("HELM_REPOSITORY_CACHE", helmpath.CachePath("repository")),
 		BurstLimit:                envIntOr("HELM_BURST_LIMIT", defaultBurstLimit),
 		QPS:                       envFloat32Or("HELM_QPS", defaultQPS),
+		LintConfigFile:            os.Getenv("HELM_LINT_CONFIG_FILE"),
```

Comment:
Can we put the default here as the other settings do? I realize we have the file path defaulted elsewhere in our implementation but this environment config module looks like it's pretty consistent about explicitly stating its defaults right here in the settings struct

Reply:
done